share/gpodder/extensions/youtube-dl.py

```diff
@@ -577,6 +581,42 @@ def toggle_embed_subtitles(self, widget):
         else:
             self.container.config.embed_subtitles = False
 
+    def check_for_update(self, widget):
+        success = False
+        try:
+            output = subprocess.check_output([sys.executable, '-m', 'pip', 'index', 'versions', program_name]).decode('utf-8')
+            logger.debug(f"pip index version {program_name}: {output}")
```

Comment:
donn't generate the debug string if not needed:
```suggestion
            logger.debug("pip index version %s: %s", program_name, output)
```